src/main/java/com/stripe/model/File.java

```diff
@@ -15,73 +17,119 @@
 @Setter
 @EqualsAndHashCode(callSuper = false)
 public class File extends ApiResource implements HasId {
-  @Getter(onMethod = @__({@Override})) String id;
-  String object;
+  /** Time at which the object was created. Measured in seconds since the Unix epoch. */
+  @SerializedName("created")
   Long created;
+
+  /** A filename for the file, suitable for saving to a filesystem. */
+  @SerializedName("filename")
   String filename;
+
+  /** Unique identifier for the object. */
+  @Getter(onMethod = @__({@Override}))
+  @SerializedName("id")
+  String id;
+
+  @SerializedName("links")
   FileLinkCollection links;
+
+  /** String representing the object's type. Objects of the same type share the same value. */
+  @SerializedName("object")
+  String object;
+
+  /**
+   * The purpose of the file. Possible values are {@code business_icon}, {@code business_logo},
+   * {@code customer_signature}, {@code dispute_evidence}, {@code finance_report_run},
+   * {@code identity_document}, {@code pci_document}, {@code sigma_scheduled_query}, or
+   * {@code tax_document_user_upload}.
+   */
+  @SerializedName("purpose")
   String purpose;
+
+  /** The size in bytes of the file object. */
+  @SerializedName("size")
   Long size;
+
+  /** A user friendly title for the document. */
+  @SerializedName("title")
   String title;
+
+  /**
+   * The type of the file returned (e.g., {@code csv}, {@code pdf}, {@code jpg}, or
+   * {@code png}).
+   */
+  @SerializedName("type")
   String type;
+
+  /** The URL from which the file can be downloaded using your live secret API key. */
+  @SerializedName("url")
   String url;
 
-  // <editor-fold desc="create">
   /**
-   * Create a file.
+   * To upload a file to Stripe, you’ll need to send a request of type {@code multipart/form-data}.
+   * The request should contain the file you would like to upload, as well as the parameters for
+   * creating a file.
+   *
+   * <p>All of Stripe’s officially supported API libraries should have support for sending
+   * {@code multipart/form-data}.
```

Comment:
This description is "quite weird" lol. I know we want to stick to the docs so not blocking but a better wording is likely worth it

Reply:
I've removed the second paragraph for now to make it slightly less weird.